css/amp.css

```diff
@@ -59,7 +59,7 @@ html.-amp-ios-embed {
 }
 
 /** Wrapper for iOS-Embed-Wrapper mode. */
-i-amp-html-wrapper {
+#i-amp-html-wrapper {
```

Comment:
this is overridable now based on current validator CSS rules right? (AFAIK there is no restriction on id selectors) Maybe we need to switch to class='-amp-html-wrapper' to make it private.